### PR TITLE
s/NoInterfaceObject/Exposed=Window, LegacyNoInterfaceObject/ in exten…

### PR DESCRIPTION
**Pull request by jdashg**
…sions/.

### CODE REVIEW

**Review by foolip — APPROVED** [review]
Yay! (Don't have write access, just expressing support.)

File: extensions/EXT_texture_compression_bptc/extension.xml [hunk 1]
```diff
@@ -75,7 +75,7 @@
     </features>
   </overview>
   <idl xml:space="preserve">
-[NoInterfaceObject]
+[Exposed=Window, LegacyNoInterfaceObject]
```
Comment (kenrussell) [inline extensions/EXT_texture_compression_bptc/extension.xml:78]: WebGL can be used on workers; throughout this needs to be:
  Exposed=(Window,Worker)
like in the WebGL 1.0 and 2.0 specs.

**Review by kenrussell — CHANGES_REQUESTED** [review]
Thanks for taking care of this; one mechanical change needed throughout.

**Review by kenrussell — APPROVED** [review]
Thanks for taking care of this; looks good!

**Pull request merged**